### ISSUE TITLE
WCAG 1.3.1,WCAG 3.3.2: Form elements must have labels (#txtcustomSearch)

### ISSUE THREAD

**Issue by hash-logic** [hash-logic/issue#5]
**Issue**: `Form elements must have labels` ([`label`](https://dequeuniversity.com/rules/axe/3.2/label?application=msftAI))

**Target application**: [Home · Custom Portal](https://mymatterssit.microsoftcrmportals.com/)

**Element path**: #txtcustomSearch

**Snippet**:

    <input type="text" title="" id="txtcustomSearch" style="margin-top:15px" placeholder="Search Profiles" data-original-title="Search">

**How to fix**:

    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Form element does not have an implicit (wrapped) <label>
      Form element does not have an explicit <label>
      Element has no title attribute or the title attribute is empty

**Environment**:
Chrome version 73.0.3683.86

====

This accessibility issue was found using Accessibility Insights for Web 2.1.2 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
